experimental/list_formatter/src/lib.rs

```diff
@@ -49,7 +49,32 @@
 extern crate alloc;
 
 mod list_formatter;
-pub mod provider;
+mod provider;
 mod string_matcher;
 
 pub use list_formatter::*;
+
+/// Represents the style of a list. See the
+/// [CLDR spec](https://unicode.org/reports/tr35/tr35-general.html#ListPatterns)
+/// for an explanation of the different styles.
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub enum ListStyle {
+    /// A typical list
+    Wide,
+    /// A shorter list
+    Short,
+    /// The shortest type of list
+    Narrow,
+}
+
+/// Each type of list uses a different [`ResourceMarker`](icu_provider::ResourceMarker).
+/// See the [CLDR spec](https://unicode.org/reports/tr35/tr35-general.html#ListPatterns)
+/// for an explanation of the different types.
+pub mod markers {
```

Comment:
thought: unsure if we really need this, but i don't feel strongly either way

Reply:
what does "this" refer to? I think we should expose these more ergonomic names. It also allows us to redirect them to `MarkerV2` in the future.

Reply:
this = the reexports

Redirecting them to a separate marker is probably a breaking change. I buy the ergonomics argument though. OTOH i do like the symmetry where all marker types have Marker in their name.